src/openforms/templates/master.html

```diff
@@ -29,6 +29,14 @@
 </head>
 
 <body class="view__body">
+    {% if enable_analytics and analytics_tools_config.gtm_code %}
```

Comment:
shouldn't this also only be emitted if Google Analyics is explicitly enabled - not only based on whether the code is filled out or not?

Reply:
This snippet should be present when the other Google Tag Manager script is present (https://github.com/open-formulieren/open-forms/blob/master/src/openforms/analytics_tools/templates/analytics_tools/google.html#L8). Unless i read it wrong, I think the other one relies on `enable_analytics` and `analytics_tools_config.gtm_code`?

Reply:
I see that the model has the field `enable_google_analytics`, but this is not used anywhere now - this should be used similar to `{% if analytics_tools_config.piwik_enabled %}` in `src/openforms/analytics_tools/templates/analytics_tools/piwik.html`

The idea is that you can configure your integration without enabling it.